src/api/package.js

```diff
@@ -83,22 +94,19 @@ export default async (providedOptions = {}) => {
       prepareSpinner.succeed();
       await packagerCompileHook(dir, ...args);
     }, async (buildPath, electronVersion, pPlatform, pArch, done) => {
-      await rebuildHook(buildPath, electronVersion, pPlatform, pArch);
-      packagerSpinner = ora('Packaging Application').start();
-      done();
-    }, async (buildPath, electronVersion, pPlatform, pArch, done) => {
       const copiedPackageJSON = await readPackageJSON(buildPath);
       if (copiedPackageJSON.config && copiedPackageJSON.config.forge) {
         delete copiedPackageJSON.config.forge;
       }
       await fs.writeFile(path.resolve(buildPath, 'package.json'), JSON.stringify(copiedPackageJSON, null, 2));
       done();
-    }].concat(forgeConfig.electronPackagerConfig.afterCopy ? forgeConfig.electronPackagerConfig.afterCopy.map(item =>
-      (typeof item === 'string' ? requireSearch(dir, [item]) : item)
-    ) : []),
-    afterExtract: forgeConfig.electronPackagerConfig.afterExtract ? forgeConfig.electronPackagerConfig.afterExtract.map(item =>
-      (typeof item === 'string' ? requireSearch(dir, [item]) : item)
-    ) : [],
+    }].concat(resolveHooks(forgeConfig.electronPackagerConfig.afterCopy, dir)),
+    afterExtract: resolveHooks(forgeConfig.electronPackagerConfig.afterExtract, dir),
+    afterPrune: [async (buildPath, electronVersion, pPlatform, pArch, done) => {
+      await rebuildHook(buildPath, electronVersion, pPlatform, pArch);
+      packagerSpinner = ora('Packaging Application').start();
```

Comment:
There's now an ora gap between "compiling" and "building native dependencies", not sure how to solve that one.

Reply:
@malept Can we just put a "Rebuilding native deps" ora before the rebuildHook runs?